Assets/Scripts/GameManager.cs

```diff
@@ -81,13 +118,24 @@ public class GameManager : MonoBehaviour {
         gameOver = (players.Count == 1);
     }
 
-    void SetHealthTexts() {
-        if (player != null) {
-            playerHealthText.text = "Health: " + player.GetComponent<TankStats>().GetStat(StatType.MaxHealth);
-        }
-        if (computerPlayer != null) {
-            computerPlayerHealthText.text =
-                "Health: " + computerPlayer.GetComponent<TankStats>().GetStat(StatType.MaxHealth);
+    void SetStatsUIs() {
+        foreach (KeyValuePair<GameObject, Text> playerStatsUI in _playerStatsUis) {
+            if (playerStatsUI.Key != null) {
+                TankHealth healthScript = playerStatsUI.Key.GetComponent<TankHealth>();
+                TankStats statsScript = playerStatsUI.Key.GetComponent<TankStats>();
+                playerStatsUI.Value.text =
+                    playerStatsUI.Key.gameObject.name + "\n" +
+                    "HP: " + healthScript.CurrentHealth + "\n" +
```

Comment:
Maybe it would be nice to show HP as `{currentHealth}/{maxHealth}`:
```
HP: 80/120
```
The `TankStats` has a `maxHealth` stat that you can retrieve